index.html

```diff
@@ -37,8 +47,8 @@ <h1> <strong>I’m Mariane
                 <div class="s-icons">
                     <ul class="media">
                         <li> <a href="https://twitter.com/alphonce_mobutu"><img class="twi" src="/photos/twitter.jpg" alt="Twitter Icon"> </a></li>
-                        <li> <a href="https://www.linkedin.com/in/adoyo-alphonce-3362a4173/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BkY43KWq1SIegum1Uq5M6dA%3D%3D"><img class="lin" src="/photos/Linkedin-icon.jpg" alt="Linkedin Icon"></a></li>
-                        <li> <a href="https://www.facebook.com/me/"><img class="mil" src="/photos/mail.jpg" alt="Mil Icon"> </a></li>
+                        <li><a href="https://www.linkedin.com/in/adoyo-alphonce-3362a4173/?lipi=urn%3Ali%3Apage%3Ad_flagship3_feed%3BkY43KWq1SIegum1Uq5M6dA%3D%3D"><img class="lin" src="/photos/Linkedin-icon.jpg" alt="Linkedin Icon"></a></li>
+                        <li><a href="https://www.facebook.com/me/"><img class="mil" src="/photos/mail.jpg" alt="Mil Icon"></a></li>
                         <li><a href="https://github.com/tingamapuro04"><img class="git" src="/photos/github.jpg" alt="github Icon"> </a></li>
                         <li> <a href="https://github.com/tingamapuro04"><img class="angel" src="/photos/angellist.jpg" alt="Angellist Icon"></a></li>
```

Comment:
- [ ] please try to change the src link as I suggest for more details check the comment above